controllers/rhmi/rhmi_controller.go

```diff
@@ -1180,24 +1182,24 @@ func (r *RHMIReconciler) processStage(installation *rhmiv1alpha1.RHMI, stage *St
 		}
 
 		// Verify that watches for this productStatus CRDs have been created
-		config, err := configManager.ReadProduct(productStatus.Name)
+		productConfig, err := configManager.ReadProduct(productName)
```

Comment:
minor nit - if we change the config read to use `productName` we should probably adjust these err messages below to use the key also, or we should change this back to retrieving from `productStatus.Name`

Reply:
Good catch !  Will revert this back to `productStatus.Name` :+1: